website/screens/components/select/specs/SelectSpecsPage.tsx

```diff
@@ -143,1056 +350,842 @@ const sections = [
+                <td>0.125rem / 2px</td>
+              </tr>
+            </tbody>
+          </DxcTable>
+        ),
+      },
+      {
+        title: "Spacing",
+        content: (
+          <>
+            <DxcParagraph>
+              The select component <strong>input</strong> share the{" "}
+              <Link href="/components/text-input/specifications" passHref>
+                <DxcLink>same spacing tokens of the text input</DxcLink>
+              </Link>
+              .
+            </DxcParagraph>
+            <DxcTable>
+              <thead>
+                <tr>
+                  <th>Property</th>
+                  <th>Element</th>
+                  <th>Token</th>
+                  <th>Value</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <td>
+                    <Code>padding-left</Code>
+                  </td>
+                  <td>List dialog</td>
+                  <td>
+                    <Code>spacing-8</Code>
+                  </td>
+                  <td>0.5rem / 8px</td>
+                </tr>
+                <tr>
+                  <td>
+                    <Code>padding-right</Code>
+                  </td>
+                  <td>List dialog</td>
+                  <td>
+                    <Code>spacing-8</Code>
+                  </td>
+                  <td>0.5rem / 8px</td>
+                </tr>
+                <tr>
+                  <td>
+                    <Code>margin-top</Code>
+                  </td>
+                  <td>List dialog content</td>
+                  <td>
+                    <Code>spacing-4</Code>
+                  </td>
+                  <td>0.25rem / 4px</td>
+                </tr>
+                <tr>
+                  <td>
+                    <Code>margin-bottom</Code>
+                  </td>
+                  <td>List dialog content</td>
+                  <td>
+                    <Code>spacing-4</Code>
+                  </td>
+                  <td>0.25rem / 4px</td>
+                </tr>
+                <tr>
+                  <td>
+                    <Code>padding-top</Code>
+                  </td>
+                  <td>List option</td>
+                  <td>
+                    <Code>spacing-4</Code>
+                  </td>
+                  <td>0.25rem / 4px</td>
+                </tr>
+                <tr>
+                  <td>
+                    <Code>padding-bottom</Code>
+                  </td>
+                  <td>List option</td>
+                  <td>
+                    <Code>spacing-4</Code>
+                  </td>
+                  <td>0.25rem / 4px</td>
+                </tr>
+              </tbody>
+            </DxcTable>
+          </>
+        ),
+      },
+      {
+        title: "Size",
+        content: (
+          <>
+            <DxcParagraph>
+              The component <Code>width</Code> can adopt the following values:
+            </DxcParagraph>
```

Comment:
I would not add this explanation to follow the rest of components documentation. I think it is clear without it.

Reply:
I think is interesting what Alex did. Since the section is called `Size` the user cannot know if this affects the width or the height. In other components, this becomes trivial since the section is called `Width`.

Reply:
After reviewing the specs of ll the components, the guidelines related to the size are explained like this:

If the section is for the `width` and `height`, the title is `Size`. If it is only the `width`, the title is called `Width`. So, for the Select component I would change the title to `Width`, remove the with explanation and the `(default)`, to keep it as the rest of the components specs.

Could you please remove also the `(default)` from Text Input? It is the only one apart from the one you added in Select.

Reply:
Agree with the Width section decision. About the `(default)`, I have already removed every appearance, so I think this is an outdated comment!